datahub-web-react/src/app/search/SearchResultsPage.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
```

Comment:
is this just a `mv` or are there changes to this file?

Reply:
Just mv my friend